.github/workflows/cts.yml

```diff
@@ -77,12 +80,10 @@ jobs:
         shell: bash
         run: |
           cd cts;
-          for backend in ${{ matrix.backends }}; do
-            echo "======= CTS TESTS $backend ======";
-            grep -v '^//' ../wgpu/cts_runner/test.lst | while IFS=$' \t\r\n' read test; do
-              echo "=== Running $test ===";
-              DENO_WEBGPU_BACKEND=$backend cargo run --manifest-path ../wgpu/cts_runner/Cargo.toml --frozen -- ./tools/run_deno --verbose "$test";
-            done
+          echo "======= CTS TESTS ======";
+          grep -v '^//' ../wgpu/cts_runner/test.lst | while IFS=$' \t\r\n' read test; do
+            echo "=== Running $test ===";
+            cargo run --manifest-path ../wgpu/cts_runner/Cargo.toml --frozen -- ./tools/run_deno --verbose "$test";
```

Comment:
Non-blocking thought - we should switch this to use `cargo xtask ctx` for consistency

Reply:
The only reason I didn't do this when I added the xtask is because this command line is additionally passing `--frozen` to cargo.

Reply:
I wouldn't worry about `--frozen` we don't use it anywhere else.